### PR TITLE
fix pop stack in preview

### PR DESCRIPTION
**Pull request by FabianDevel**

### CODE REVIEW

File: app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt [hunk 1]
```diff
@@ -215,7 +215,9 @@ class FileInfoActionsView @JvmOverloads constructor(
                     availableOfflineSwitch.isChecked = false
                     showSnackBarInvalidFileName(currentFile.name)
                 }
-                findNavController().popBackStack()
+                with(findNavController()) {
+                    if (currentBackStackEntry?.destination?.label == "FileInfoActionsBottomSheetDialog") popBackStack()
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt:219]: Can you put this `FileInfoActionsBottomSheetDialog` in the `strings.xml`, as a non-translatable one?

And use it too in the `main_navigation.xml`.

Comment (FabianDevel) [reply app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt]: done

File: app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt [hunk 2]
```diff
@@ -215,7 +215,11 @@ class FileInfoActionsView @JvmOverloads constructor(
                     availableOfflineSwitch.isChecked = false
                     showSnackBarInvalidFileName(currentFile.name)
                 }
-                findNavController().popBackStack()
+                with(findNavController()) {
+                    if (currentBackStackEntry?.destination?.label == getString(R.string.fileActionBottomSheetDialog)) {
```
Comment (JorisBodin) [inline app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt:219]: Use `currentBackStackEntry?.destination?.id`

**Review by JorisBodin — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt (hunk 2 above)
Comment (FabianDevel) [reply app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt]: done

**Review by JorisBodin — APPROVED** [review]

**Pull request merged**